server/models/rpaTaskModel.test.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-undef */
+const mongoose = require('mongoose');
+const { expect } = require('chai');
+const dbHandler = require('../utils/TestingUtils/testDatabaseHandler.js');
+const { testRpaTask1 } = require('../utils/TestingUtils/testData.js');
+const taskModel = require('./rpaTaskModel.js');
```

Comment:
Did we need this `taskModel`? ESLint throws an error. So please use `disable no-unused-vars` for this line if it is really needed.

Reply:
yes, it's needed. Disabled it there